src/binutil/server.rs

```diff
@@ -344,7 +344,8 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
     let mut status_enabled = cfg.metric.address.is_empty() && !server_cfg.status_addr.is_empty();
 
     // Create a status server.
-    let mut status_server = StatusServer::new(server_cfg.status_thread_pool_size);
+    // TODO: How to keep cfg updated?
```

Comment:
@siddontang Does tikv support internal config update without needing to restart itself?

Reply:
This feature *will* be added soon.

Reply:
Is there any planned way to handle replicating config changes through the modules? Destroy/recreate the status server seems a bit extreme.

Reply:
I think the solution would be a (global) Config repository that provides config access and mutation interfaces, maybe change listeners as well.

Reply:
I think the solution would be a (global) Config repository that provides config access and mutation interfaces, maybe change listeners as well.